custom-vu/src/main/kotlin/jces1209/vu/JiraDcScenario.kt

```diff
@@ -55,11 +54,6 @@ class JiraDcScenario : Scenario {
                 projectMemory = similarities.projectMemory,
                 createIssueButtons = listOf(By.id("create_link"))
             ),
-            searchWithJql = SearchServerFilter(
```

Comment:
These changes are already in #69. Combining multiple changes in a single PR makes it messy. Reviewers have to review the same diffs multiple times. When one gets merged the other one has to rebase/resolve conflicts. The multi-change diff is huge and hard to understand.